techniques/general/G58.html

```diff
@@ -131,9 +125,9 @@ <h3>Example 1: A .MOV Document in an HTML Document</h3>
 &lt;/p&gt;</code></pre>
 		</section>
 		<section class="example" id="example-2-the-link-back-to-the-.mov-document-in-an-html-document">
-			<h3>Example 2: The link back to the .MOV Document in an HTML Document</h3>
+			<h3>事例 2: HTML ドキュメント内の .MOV ドキュメントへ戻る</h3>
```

Comment:
The link が訳出されていないので追記しました。
```suggestion
			<h3>事例 2: HTML ドキュメント内の .MOV ドキュメントへ戻るリンク</h3>
```